orchagent/muxorch.cpp

```diff
@@ -841,8 +928,44 @@ void MuxOrch::updateNeighbor(const NeighborUpdate& update)
         if (ptr->isIpInSubnet(update.entry.ip_address))
         {
             ptr->updateNeighbor(update.entry, update.add);
+            return;
```

Comment:
Q: Just for my understanding here. So when this function is entered, and the update ip address is in Subnet and is already in the mux_cable_tb, we just need to updateNeighbor and nothing else needs to be handled?

Reply:
thats right.. because we already know the mux cable and doesn't have to look fdb for it